sections/compare-blender.liquid

```diff
@@ -1,17 +1,366 @@
+        var tempHighest = highest.slice();
+        setCellHeight(grid['childNodes'][i], tempHighest);
+      }
+    }
+  }
+
+  var getTallestHeight = function(grid) {
+    var highest = [];
+    for (var i in grid['childNodes']) {
+      if (grid['childNodes'][i] && grid['childNodes'][i].classList && grid['childNodes'][i].classList.contains('grid__item')) {
+        getHeightOfEachCell(grid['childNodes'][i], [], highest);
+      }
+    }
+    return highest;
+  };
+
+  var standardize_cell_size = function() {
+    var grids = document.getElementsByClassName('grid');
+    for (var i = 0; i < grids.length; i++) {
+      var arrOfHeights = [];
+      var highest = getTallestHeight(grids[i]);
+      setCellHeightByGridItem(grids[i], highest);
+    }
+  };
+
+  var makeOnlyLeftMostInfoVisible = function() {
+    var sections = document.getElementsByClassName('blender-info-side');
+    for (var i in sections) {
+      if (sections[i] && sections[i].style) {
+          sections[i].style.display = null;
+          var parentLeftDiff = sections[i].parentElement.offsetLeft;
+        if (sections[i].offsetLeft - parentLeftDiff > 25) {
+          sections[i].style.display = 'none';
+        } else {
+          sections[i].style.display = null;
+        }
+      }
+    }
+  };
+
+  var zoomin = function(product) {
+    var index = productSelectedVariant[product];
+
+    document.getElementById('popupBackground').style.display = null;
+    document.getElementById('thePopup').style.display = null;
+    document.getElementById('zoomInProduct').src = variantToOptions[index].src;
+    document.getElementById('zoomInPanel').src = variantToOptions[index].panel;
+  };
+
+  var exitPopup = function() {
+    document.getElementById('popupBackground').style.display = 'none';
+    document.getElementById('thePopup').style.display = 'none';
+  };
+
+standardize_cell_size();
+makeOnlyLeftMostInfoVisible();
+window.addEventListener("resize", function() {
+ standardize_cell_size();
+ makeOnlyLeftMostInfoVisible();
+});
+</script>
+
+
+<style>
+.product-comparison--bg-image{
+    background-image: url('{{ section.settings.top_image | img_url: '1000x' }}}');
+    background-position: 0 50%;
+    background-size: cover;
+    display: inline-block;
+    width: 100%;
+}
+.product-comparison--header-container {
+    padding: 100px 0 120px;
+}
+.product-comparison--sub-leader {
+    font-size: 28px;
+    font-size: 2rem;
+    color: #fff;
+    text-align: center;
+    text-transform: uppercase;
+}
+.product-comparison--leader {
+    font-size: 56px;
+    font-size: 4rem;
+    color: #fff;
+    text-align: center;
+    text-transform: uppercase;
+}
+@media screen and (min-width: 980px) /* Desktop */ {
+  #thePopup{
+    width:60%;
+    left: 20%;
+  }
+}
+
+@media screen  and (max-width: 979px) /* Tablet */ {
+  #thePopup{
+    width:80%;
+    left: 10%;
+  }
+
+}
+
```

Comment:
I think we can piggy back on the shopify modal style I forget the elements but snagged them from the newsletter popup.